### ISSUE TITLE
Failed opening required upgradelib.php

### ISSUE THREAD

**Issue by mudrd8mz** [go1com/moodle-mod_goone#1]
https://github.com/go1com/moodle-mod_goone/blob/afe770bc0c6813b3acafec1152d1d5d359bf7edb/db/upgrade.php#L28

requires a non-exisiting file. As a result, attempting to upgrade from previous versions makes the upgrade process fail with fatal error:

```
Fatal error: require_once(): Failed opening required '.../mod/goone/db/upgradelib.php' in
.../mod/goone/db/upgrade.php on line 28
```

**Comment by mudrd8mz** [comment]
This does not affect fresh new installations. But it would become a serious blocker for all future upgrades. Please fix ASAP.

**Comment by mudrd8mz** [comment]
The best solution for this as well as for the bug #2 would be to simply remove the current db/upgrade.php

Ping @fouad-ecreators

**Comment by fouad-ecreators** [comment]
@mudrd8mz Resolved this as well as #2, was also possibly caused by the skeleton generator.
